lib/linter.js

```diff
@@ -74,12 +74,19 @@ module.exports = function (cwd, args, text) {
 
     return JSON.stringify(fileConfig, null, '  ');
   }
+  if (currentOptions.fixToStdout && !stdin) {
```

Comment:
Hm, I see. Checking for `text` shouldn't be necessary. I agree that the flag should be enough.